config/default.json

```diff
@@ -100,5 +100,6 @@
   "forbiddenTypesForParallelIngesion": [
     "Ingestion_New",
     "Ingestion_Update"
-  ]
+  ],
+  "resolutionTolerance": 12
```

Comment:
move to "validationValuesByInfo" section

Reply:
fixed